pkg/capabilities/capabilities.go

```diff
@@ -11,7 +11,8 @@ import (
 	"kernel.org/pub/linux/libs/security/libcap/cap"
 )
 
-var Caps Capabilities // singleton for all packages
+var once sync.Once
+var Caps *Capabilities // singleton for all packages
```

Comment:
Let's keep Caps unexported and rename NewCapabilities to GetInstance as we already discussed